docker-for-aws/persistent-data-volumes.md

```diff
@@ -22,44 +36,88 @@ f416c95c0dcc        cloudstor:aws               cloud storage plugin for Docker
 
 The following examples show how to create swarm services that require data persistence using the --mount flag and specifying Cloudstor as the driver.
 
-### Share the same volume between tasks:
+### Share the same volume between tasks (EFS support required and enabled):
+
+In those regions where EFS is supported and EFS support is enabled during deployment of the Cloud Formation template, `shared` Cloudstor volumes can be created to share access to persistent data across all tasks in a swarm service running in multiple nodes. Example usage:
 
 ```bash
 docker service create --replicas 5 --name ping1 \
     --mount type=volume,volume-driver=cloudstor:aws,source=sharedvol1,destination=/shareddata \
     alpine ping docker.com
 ```
 
-Here all replicas/tasks of the service `ping1` share the same persistent volume `sharedvol1` mounted at `/shareddata` path within the container. Docker Swarm takes care of interacting with the Cloudstor plugin to make sure the common backing store is mounted on all nodes in the swarm where tasks of the service are scheduled. Each task needs to make sure they don't write concurrently on the same file at the same time and cause corruptions since the volume is shared.
+All replicas/tasks of the service `ping1` above share the same persistent volume `sharedvol1` mounted at `/shareddata` path within the container. Docker Swarm takes care of interacting with the Cloudstor plugin to make sure EFS is mounted on all nodes in the swarm where tasks of the service are scheduled. Each task needs to make sure they don't write concurrently on the same file at the same time as that may lead to data corruption since the volume is shared.
 
-With the above example, you can make sure that the volume is indeed shared by logging into one of the containers in one swarm node, writing to a file under `/shareddata/` and reading the file under `/shareddata/` from another container (in the same node or a different node).
+In the above example, you can make sure that the volume is indeed shared by logging into one of the containers in one Swarm node, writing to a file under `/shareddata/` and reading the file under `/shareddata/` from another container (in a different node).
 
-### Use a unique volume per task:
+The only option available for EFS currently is `perfmode`. If high IO throughput is desired, please set the `perfmode` volume-opt to `maxio`. Example:
 
 ```bash
 {% raw %}
-docker service create --replicas 5 --name ping2 \
-    --mount type=volume,volume-driver=cloudstor:aws,source={{.Service.Name}}-{{.Task.Slot}}-vol,destination=/mydata \
+docker service create --replicas 5 --name ping3 \
+    --mount type=volume,volume-driver=docker4x/cloudstor:aws-v{{ edition_version }},source={{.Service.Name}}-{{.Task.Slot}}-vol5,destination=/mydata,volume-opt=perfmode=maxio \
```

Comment:
Since we alias the plugin to prevent upgrade failures, shouldn't this `volume-driver` remain `cloudstor:aws`?